### PR TITLE
Adds two new pages for sending notifications with messages and NOR co…

### PR DESCRIPTION
**Pull request by humancompanion-usds**
…des. [Amends #649]

The only thing I'm lacking is the NOR codes for those moving forward.

### CODE REVIEW

File: pages/phases/phone-assessment-interviews/notifications.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+layout: article
+permalink: /hiring-phases/phone-assessment-interviews/notifications/
+section: hiring-phases
+title: Sending Notifications Post Phone Assessment Interview
+sidenav: hiring-phases
+phase: phone-assessment-interviews
+sub-phase-order: 040
+intro: It is appropriate to send notifications to all applicants at the end of each phase. Use the following NOR codes and notification messages to communicate with applicants.
```
Comment (bernars-usa) [inline pages/phases/phone-assessment-interviews/notifications.md]:
```suggestion
intro: As a best practice, you should send notifications to all applicants at the end of each phase. Use these NOR codes and notification messages to communicate with applicants.
```

**Comment by humancompanion-usds** [comment]
@bernars-usa - Yes, but my understanding is that those are new codes, which is great, but they on the USA Staffing backlog and thus don't exist yet. Do we want to add them now and indicate they are future codes? I was holding off on that but, perhaps it would be useful for HR folks to know to A. Understand we are working with USAS on these items and B. So they can request status on them from USAS?

**Comment by bernars-usa** [comment]
> Yes, but my understanding is that those are new codes, which is great, but they on the USA Staffing backlog and thus don't exist yet.

Oh, yes, to be clear, I was just dumping them here so you knew they were in progress. https://github.com/usds/OPM_pilot/issues/600 didn't have that information, and the issue was closed. I'm not sure how far along Maria is with implementation.

> perhaps it would be useful for HR folks to know to A. Understand we are working with USAS on these items and B. So they can request status on them from USAS?

Eh, I don't know. The email messages as-is can be implemented as custom notifications in USAS right now without the codes, right? I don't like to give future-looking statements in docs. Anything can happen in a backlog.

**Comment by humancompanion-usds** [comment]
@bernars-usa - Okay, I added the future NOR codes into the page but commented out. Hopefully that will keep us from forgetting they are there. Is this otherwise good to go? Are you okay with the additional page for each section specifically for notifications?

**Comment by bernars-usa** [comment]
> Are you okay with the additional page for each section specifically for notifications?

I don't love it, but we don't have a better architecture right now, so it's ok.

I made some changes to the titles and shortdescs. I think this is good to go at this point. 👍

**Pull request merged**